Python/Code_Template.md

```diff
@@ -0,0 +1,49 @@
+# Python Coding Template 
+
+Here is the general code template a contributor must follow in order to maintain the consistency and quality of the codes.The template is comprised of many individual components which may be multi-lined comments or code. The components of the template is briefly discussed bellow.
+
+
+## Introduction:
+
+        This contains the basic description of the algorithm used or the problem the code focuses to solve. The description should be brief consisting of not more than 3-5 lines. In case of a problem description, one should consider 
+giving at least one example.
+
+## Purpose:
+
+        This portion of the code deals with the basic purpose of the code, i.e the desired operation the code is performing. This section can contain only the name of the algorithm without much description. This part will be small and comprise of 2-3 lines.
+
+## Method:
+
+        One-line comment contains the name of the algorithm/ technique/ data structure used to solve the given problem.
+
+## Time / Space Complexity:
+
+        As the name signifies, it contains the worst time/ space complexity of the solution/ algorithm used in Big-O notation.
+
+## Argument / Return Type:
+
+        The data type of the argument and the return value of the main function involved in the solution or implementation of the algorithm is specifically provided.
+
+## Code / Solution:
+
+        This is the only un-commented part of the template where you write the implementation of the algorithm or the solution of a problem discussed earlier. The code should be formatted to PEP8 standard guideline The variable name should be meaningful and the function name should be such that it signifies its usage in the program. Important lines of implementation of the algorithm and variable declaration should be backed with a relatable single-line comment which justifies the implementation and necessary for the variable declaration.
```

Comment:
Can you please change this paragraph into points ? As it would help contributors to easily check if their code is even with the expected template.

Reply:
Sure

Reply:
@atarax665 the request changes are made, you can review it again